README.md

```diff
@@ -66,6 +66,7 @@ New methods are added to `Map.prototype`.
   * `Map.prototype.filter(predicate, thisArg)`
   * `Map.prototype.mapValues(fn, thisArg)`
   * `Map.prototype.mapKeys(fn, thisArg)`
+  * `Map.prototype.reduce(fn, initialValue)`
```

Comment:
what about reduceRight?

Reply:
The current spec iterates with "For each Record { [[Key]], [[Value]] } e that is an element of entries, in original key insertion order" - I'm sure we could say "For each Record { [[Key]], [[Value]] } e that is an element of entries, in the reverse of original key insertion order" or similar.

Reply:
I mean another way, not in this method. In userland, we can't just get the last item of a map without passing all element, we have just iterators and `.forEach` and they work in the order of insertion. So I'm not sure that iteration in the reverse order makes sense for maps / sets conception.

Reply:
I don't think the ease of polyfilling is particularly important; a polyfill can iterate everything, store it, and then iterate over it in reverse.

Reply:
It's not related polyfilling. It's adding completely new conception to collections. And the first part of this conception - reverse iterators. Anyway, it should be discussed in another issue and I'm not sure that it will be accepted - even usual `.reduce` had too many objections.

Reply:
Fair point.